nemo/collections/nlp/models/language_modeling/megatron_t5_speechlm_model.py

```diff
@@ -124,14 +156,6 @@ def __init__(self, cfg: DictConfig, trainer: Trainer):
                 self.frozen_model.enc_dec_model.decoder_cfg.hidden_size, speech_codebook_size
             )
 
-        self.speech_offset = speech_offset
```

Comment:
Why was all of these initializations removed from here?

Reply:
The initializations have been moved above (Line 104 to Line 110). I believe some of the variables are being used in initializing speech heads thats why they were moved above.